CMakeLists.txt

```diff
@@ -56,6 +56,15 @@ install(DIRECTORY cmake DESTINATION share/cartographer/)
 file(GLOB_RECURSE ALL_LIBRARY_HDRS "*.h")
 file(GLOB_RECURSE ALL_LIBRARY_SRCS "*.cc")
 file(GLOB_RECURSE ALL_TESTS "*_test.cc")
+
+# Remove dotfiles/-folders that could potentially pollute the build.
+file(GLOB_RECURSE ALL_DOTFILES ".*/*")
+if (ALL_DOTFILES)
+  list(REMOVE_ITEM ALL_LIBRARY_HDRS ${ALL_DOTFILES})
+  list(REMOVE_ITEM ALL_LIBRARY_SRCS ${ALL_DOTFILES})
+  list(REMOVE_ITEM ALL_TESTS ${ALL_DOTFILES})
```

Comment:
apply remove_item also to ALL_EXECUTABLES?

Reply:
done